tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -50,9 +51,10 @@ public void process(final Socket connection) {
              final var outputStreamWriter = new OutputStreamWriter(connection.getOutputStream())) {
             InputView inputView = new InputView(new BufferedReader(inputStreamReader));
             HttpRequest httpRequest = readHttpRequest(inputView);
+            HttpResponse httpResponse = HttpResponse.status(HttpStatus.OK).build();
```

Comment:
```suggestion
            HttpResponse httpResponse = HttpResponse.ok();
```

쓰는 사람 입장에서는 이게 더 편리하지 않을까요?